### PR TITLE
centos7 init scripts

### PR DESCRIPTION
**Pull request by jburel**
Point to the service.d scripts. 
This might require a quick release of the doc. 
cc @hflynn @sbesson @joshmoore

### CODE REVIEW

File: omero/sysadmins/unix/server-centos7.txt [hunk 1]
```diff
@@ -165,15 +165,15 @@ Nginx or Apache should already be running so you should be able to log in as the
 root user by going to http://localhost/ in your web browser.
 Please read the SELinux_ section below.
 
-In addition some example `init.d` scripts are available should you wish to
+In addition some example `systemd.service` scripts are available should you wish to
 start OMERO and OMERO.web automatically:
 
-.. literalinclude:: walkthrough/step06_centos6_daemon.sh
```
Comment (joshmoore) [inline omero/sysadmins/unix/server-centos7.txt:171]: Urgh. Good catch!

**Comment by pwalczysko** [comment]
Sorry, did not manage today. Will test tomorrow.

**Comment by pwalczysko** [comment]
The script is replaced in http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/unix/server-centos7.html as intended here. Will run through the workflow yet in the afternoon.

**Comment by jburel** [comment]
@pwalczysko: I don't think it is required to go over a full installation.

**Comment by pwalczysko** [comment]
Okay, checked that the script content matches the text in the doc itself above. Ready to merge.

**Comment by hflynn** [comment]
Will build and release this now given the blooper with linking to the wrong script.

**Pull request merged**